catalog/ui/search-ui/standard/pom.xml

```diff
@@ -146,23 +146,23 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>com.github.eirslett</groupId>
-                <artifactId>frontend-maven-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>grunt test</id>
-                        <goals>
-                            <goal>grunt</goal>
-                        </goals>
-                        <phase>test</phase>
-                        <configuration>
-                            <skip>${skipTests}</skip>
-                            <arguments>test</arguments>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
+            <!--<plugin>-->
```

Comment:
We can't comment out the entire frontend plugin. The UI won't even get built with this change. We need to dive into the npm dependencies themselves and figure out which ones are failing and leave those out of the build.